types/audit-details.d.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+  module LH.Audit.Details {
+    export interface Filmstrip {
+      type: 'filmstrip';
+      scale: number;
+      items: {
+        timing: number;
+        timestamp: number;
```

Comment:
wanna throw a comment on these bad boys for filmstrip? we were inconsistent when we added the full page screenshot so documenting what they represent seems like a decent idea :)

Reply:
> wanna throw a comment on these bad boys for filmstrip

done